On The Map/Controller/LoginViewController.swift

```diff
@@ -0,0 +1,83 @@
+//
+//  LoginViewController.swift
+//  On The Map
+//
+//  Created by Michal Hus on 2/20/20.
+//  Copyright © 2020 Michal Hus. All rights reserved.
+//
+
+import UIKit
+
+class LoginViewController: UIViewController {
+    
+    var students:[Student] = []
+    let loginErrorTitle:String = "Log In Failure"
+    
+    @IBOutlet weak var emailTextField: UITextField!
+    @IBOutlet weak var passwordTextField: UITextField!
+    @IBOutlet weak var loginButton: UIButton!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+    }
```

Comment:
No need to override if all you do is call `super.whatever()`. If you delete this code it will have the exact same behavior.

Reply:
I think I might have forgotten to delete it.
Like you said its definitely good practice to review your own code.